common/cache/configurer.go

```diff
@@ -28,7 +28,7 @@ func Get(config *config.Config) Cache {
 var cacheFactory CacheFactory
 
 func init() {
-	if diskCachePath := os.Getenv("ASPECT_CONFIGURE_CACHE"); diskCachePath != "" {
```

Comment:
This is non-public, although I know of a few devs using it who I'll have to inform.